autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -84,6 +84,12 @@
     <include file="$(find-pkg-share web_controller)/launch/web_controller.launch.xml" unless="$(var scenario_simulation)"/>
   </group>
 
+  <!-- API -->
+  <include file="$(find-pkg-share tier4_autoware_api_launch)/launch/autoware_api.launch.xml">
+    <arg name="init_simulator_pose" value="true"/>
+    <arg name="init_localization_pose" value="false"/>
+  </include>
+
```

Comment:
https://github.com/tier4/autoware_launch/blob/d629633e3c9533fb54e715eca29a6c0aded56661/autoware_launch/launch/planning_simulator.launch.xml#L69-L75

Reply:
For planning simulator, it seems like the booleans are matching with the original code. (and it seems to match with what @isamu-takagi said in https://github.com/autowarefoundation/autoware_launch/pull/26#issuecomment-1093877404)

Reply:
Yes, I'll resolve this comment.